codes/javascript/chapter_searching/binary_search_edge.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * File: binary_search_edge.js
+ * Created Time: 2023-06-04
+ * Author: Justin (xiefahit@gmail.com)
+ */
+
+function binary_search_left_edge(nums, target) {
+    /* 二分查找最左一个元素 */
```

Comment:
The name should be placed at the top of the function.

Reply:
OK